test/build.bats

```diff
@@ -137,6 +137,31 @@ load common
     rm -Rf --one-file-system "$tar" "$img"
 }
 
+@test 'ch-pull2tar' {
+    scope standard
+    need_docker
+    tag='alpine:latest'
+    tar="${ch_tardir}/${tag}.tar.gz"
+    ch-pull2tar "$tag" "$ch_tardir"
+    [[ $status -eq 0 ]]
+    [[ -e $tar ]] 
+    rm "${ch_tardir}/${tag}.tar.gz"
+    [[ ! -e $tar ]]
+}
+
+@test 'ch-pull2dir' {
+    scope standard
+    # This test unpacks in $ch_tardir and is removed upon completion.
+    need_docker
+    tag='alpine:latest'
+    img="${ch_tardir}/${tag}"
+    ch-pull2dir "$tag" "$ch_tardir"
+    [[ status -eq 0 ]]
+    [[ -d $img ]]
+    rm -Rf --one-file-system "$img"
```

Comment:
So I'm a bit queasy that this just deletes the directory image instead of testing it, but `ch-build2dir` has the same problem. Maybe a new issue to test both.

Reply:
I agree. We should probably do a `ch-run $img -- /bin/true` at the very least. That said, I think it should probably be a new issue to address both tests.